src/locales.ts

```diff
@@ -0,0 +1,136 @@
+import { Translations } from "./components/providers/localeProvider"
+
+export const locales = ["en_US", "uk_UA"]
+
+export const translations : Translations = {
+    en_US: {
+        layout: {
+            home: "Home",
+            cars: "Cars",
+            moto: "Motorcycles",
+            list: "List",
+            lang_name: "🇬🇧 English",
+        },
+        form: {
+            select_maker: "Select maker",
+            maker: "Maker",
+            model: "Model",
+            year: "Year",
+            engine: "Engine",
+            engine_text: "Engine Type",
+            petrol: "Petrol",
+            diesel: "Diesel",
+            electric: "Electric",
+            save: "Save",
+            cancel: "Cancel",
+            reset: "Reset",
+            commit: "Commit",
+            commit_hist: "Commits History",
+            cars: "Cars",
+            editcar: "Edit Car",
+            editmoto: "Edit Motorcycle",
+            type: "Type",
+            favorite: "Favorite",
+            vehicle_list: "Vehicle List"
+        },
+        error: {
+            maker_error: "Maker is required",
+            model_error: "Model is required",
+            model_5ch_error: "Model must be at least 5 characters long",
+            model_2ch_error: 'Model must be at least 2 characters long',
+            model_15ch_error: "Model must be at most 15 characters long",
+            year_error: "Year is required",
+            year_error_1900: "Year must be greater than 1900",
+            year_less_error: "Year must be less than ",
+            year_4num_error: "Year must be a 4 digit number",
+            engine_error: "Engine type is required"
+        },
+        car_form: {
+            title: "Add car",
+            submit: "Add Car",
+            clear: "Clear",
+        },
+        moto_form: {
+            title: "Add motorcycle",
+            submit: "SUBMIT",
+            clear: "CLEAR",
+        },
+        page_not_found: {
+            title: "404 Page not found",
+            message: "Sorry, the page you are looking for could not be found or has been removed."
+        },
+        page_home: {
+            title: "Your own collection",
+            message: "Our team is working on a web project for collecting vehicles. This project allows you to create collections of your own cars and motorcycles.",
+            message1: "Vladyslav made a form for creating a car collection.",
+            message2: "Irina created a form to create a collection of motorcycles.",
+            message3: "To connect these two forms, we used the react-router-dom library.",
+            message4: "",
```

Comment:
Do you need to have separate messages for each sentence? maybe it can be in a single string?
(you can style typography to wrap it as needed)